### PR TITLE
Fix organization's profile picture wording

### PR DESCRIPTION
**Pull request by loic-sharma**
Current wording:

> We use the verified email address for your organization and gravatar.com to get the publicly visible profile picture. gravatar.com to change the profile picture for your organization.

Fixed wording:

> We use the verified email address for your organization and gravatar.com to get the publicly visible profile picture. Go to gravatar.com to change the profile picture for your organization.

Fixes https://github.com/NuGet/NuGetGallery/issues/6096

### CODE REVIEW

File: src/NuGetGallery/Views/Shared/_AccountProfilePicture.cshtml [hunk 1]
```diff
@@ -6,7 +6,7 @@
     {
         ViewBag.UnconfirmedEmailAddressMessage = "This is a preview of how the profile picture will look once the organization email address is verified.";
         ViewBag.GravatarProfileMessage = "We use the verified email address for your organization and {0} to get the publicly visible profile picture. "
-            + "{0} to change the profile picture for your organization.";
+            + "{1} to change the profile picture for your organization.";
```
Comment (agr) [inline src/NuGetGallery/Views/Shared/_AccountProfilePicture.cshtml]: I don't see the `Go to` part added as suggested by PR description.

Comment (agr) [reply src/NuGetGallery/Views/Shared/_AccountProfilePicture.cshtml]: Should it be "Visit gravatar.com" instead?

Comment (agr) [reply src/NuGetGallery/Views/Shared/_AccountProfilePicture.cshtml]: Oh, I see, "Go to" is inserted when formatting. That's weird way to do that... Should we have `gravatar.com` as a link?

Comment (loic-sharma) [reply src/NuGetGallery/Views/Shared/_AccountProfilePicture.cshtml]: Agreed, fixed!

**Pull request merged**